.jules/instructions.md

```
@@ -1,5 +1,5 @@
 # Instruções Operacionais para Jules — DJ Zen Eyer
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)